app/views/posts/index.html.erb

```diff
@@ -10,6 +10,6 @@
     Recent posts
   </h3>
   <ul class="posts">
-    <%= render @timeline_posts %>
+    <%= render @friends %>
```

Comment:
:bulb: This line is raising an error. I needed to comment it out to make the application work.
`<%= render @friends %>`

![image](https://user-images.githubusercontent.com/60448833/96029547-7dcbee00-0e63-11eb-8660-463586822535.png)